Wabbajack.App.Blazor/App.xaml.cs

```diff
@@ -30,30 +33,43 @@ private static void SetupLogging(ILoggingBuilder loggingBuilder)
 
         var fileTarget = new FileTarget("file")
         {
-            FileName = "log.log"
+            FileName = "logs/Wabbajack.current.log",
+            ArchiveFileName = "logs/Wabbajack.{##}.log",
+            ArchiveOldFileOnStartup = true,
+            MaxArchiveFiles = 10,
+            Layout = "${processtime} [${level:uppercase=true}] (${logger}) ${message:withexception=true}",
```

Comment:
Continuing our heated discussion from the Discord: pipes are nice `"${processtime}|${level:uppercase=true}|${logger}|${message:withexception=true}"` which is just the default NLog layout but `processtime` instead of `longdate`.

Reply:
😠